lib/api/objectPutCopyPart.js

```diff
@@ -122,29 +127,86 @@ function objectPutCopyPart(authInfo, request, sourceBucket,
                         copyLocator.copyObjectSize);
                 });
         },
-        function checkMPUBucketAuth(dataLocator, destBucketMD,
```

Comment:
The services.metadataValidateMultipart function tries to do too much and then doesn't do something we need to do here now-- get from the mpu overview object the controlling location for the data for the mpu.  We are still using the services.metadataValidateMultipart in a few other api's where it fits the need though it could certainly be refactored at some point.